lib/dropbox_api/endpoints/files/copy_batch_check.rb

```diff
@@ -0,0 +1,22 @@
+module DropboxApi::Endpoints::Files
+  class CopyBatchCheck < DropboxApi::Endpoints::Rpc
+    Method      = :post
+    Path        = "/2/files/copy_batch/check_v2".freeze
+    ResultType  = DropboxApi::Results::CopyBatchCheckResult
+    ErrorType   = DropboxApi::Errors::PollError
+
+    # 
+    # @param async_job_id [String] Id of the asynchronous job. 
+    #   This is the value of a response returned from the method that launched the job.
+    # 
+    # @return Returns the status of an asynchronous job for copy_batch:2. 
+    # It returns list of results for each entry.  
+    #
+
```

Comment:
Could you update the documentation of the endpoint?

Normally I just copy the wording from Dropbox API docs, then adjust the bits that need to so it makes sense for the person who'll be using this library. Finally trim the lines that exceed 80 cols.

Reply:
Updating it :)